<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CLocatedBindablePage</class>
 <widget class="QWidget" name="CLocatedBindablePage">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>262</width>
    <height>473</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_10">
   <property name="horizontalSpacing">
    <number>5</number>
   </property>
   <property name="margin">
    <number>6</number>
   </property>
   <item row="0" column="0">
    <widget class="QTabWidget" name="tabWidget">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="tabPosition">
      <enum>QTabWidget::East</enum>
     </property>
     <property name="currentIndex">
      <number>0</number>
     </property>
     <property name="elideMode">
      <enum>Qt::ElideNone</enum>
     </property>
     <property name="tabsClosable">
      <bool>false</bool>
     </property>
     <property name="movable">
      <bool>true</bool>
     </property>
     <widget class="QWidget" name="materialPage">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <attribute name="title">
       <string/>
      </attribute>
      <layout class="QGridLayout" name="gridLayout_2">
       <property name="margin">
        <number>6</number>
       </property>
       <property name="spacing">
        <number>3</number>
       </property>
       <item row="0" column="0">
        <layout class="QHBoxLayout" name="horizontalLayout_2">
         <item>
          <widget class="QLabel" name="blendModeLabel">
           <property name="text">
            <string>Blending mode:</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QComboBox" name="blendModeComboBox">
           <item>
            <property name="text">
             <string>Add</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>Modulate</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>Alpha blend</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>Alpha test</string>
            </property>
           </item>
          </widget>
         </item>
        </layout>
       </item>
       <item row="1" column="0">
        <layout class="QHBoxLayout" name="horizontalLayout_3">
         <item>
          <widget class="QCheckBox" name="autoLodCheckBox">
           <property name="text">
            <string>No AutoLOD</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QCheckBox" name="globalColorLightingCheckBox">
           <property name="text">
            <string>Global color lighting</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item row="2" column="0">
        <widget class="QCheckBox" name="ztestCheckBox">
         <property name="text">
          <string>Z-Test</string>
         </property>
        </widget>
       </item>
       <item row="3" column="0">
        <layout class="QHBoxLayout" name="horizontalLayout">
         <item>
          <widget class="QLabel" name="zbiasLabel">
           <property name="text">
            <string>ZBias:</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QDoubleSpinBox" name="zbiasDoubleSpinBox">
           <property name="minimum">
            <double>-9999.000000000000000</double>
           </property>
           <property name="maximum">
            <double>9999.000000000000000</double>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item row="4" column="0">
        <widget class="NLQT::CAttribRGBAWidget" name="colorWidget">
         <property name="title">
          <string>Color</string>
         </property>
        </widget>
       </item>
       <item row="5" column="0">
        <spacer name="verticalSpacer">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>21</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="sizeAnglePage">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <attribute name="title">
       <string/>
      </attribute>
      <layout class="QGridLayout" name="gridLayout_4">
       <property name="margin">
        <number>6</number>
       </property>
       <property name="spacing">
        <number>3</number>
       </property>
       <item row="0" column="0">
        <widget class="QCheckBox" name="independentSizeCheckBox">
         <property name="text">
          <string>Independent size</string>
         </property>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QGroupBox" name="independentGroupBox">
         <property name="enabled">
          <bool>false</bool>
         </property>
         <property name="title">
          <string/>
         </property>
         <layout class="QGridLayout" name="gridLayout_3">
          <property name="margin">
           <number>9</number>
          </property>
          <property name="spacing">
           <number>6</number>
          </property>
          <item row="0" column="1">
           <widget class="QRadioButton" name="heightRadioButton">
            <property name="text">
             <string>Height</string>
            </property>
           </widget>
          </item>
          <item row="0" column="0">
           <widget class="QRadioButton" name="widthRadioButton">
            <property name="text">
             <string>Width</string>
            </property>
            <property name="checked">
             <bool>true</bool>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item row="2" column="0">
        <widget class="NLQT::CAttribFloatWidget" name="sizeWidget">
         <property name="title">
          <string>Size</string>
         </property>
        </widget>
       </item>
       <item row="3" column="0">
        <widget class="NLQT::CAttribFloatWidget" name="angle2DWidget">
         <property name="title">
          <string>Angle 2D</string>
         </property>
        </widget>
       </item>
       <item row="4" column="0">
        <spacer name="verticalSpacer_2">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>31</width>
           <height>354</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="rotatePage">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <attribute name="title">
       <string/>
      </attribute>
      <layout class="QGridLayout" name="gridLayout_7">
       <property name="margin">
        <number>6</number>
       </property>
       <property name="spacing">
        <number>3</number>
       </property>
       <item row="0" column="0">
        <widget class="NLQT::CAttribPlaneBasisWidget" name="particlePlaneBasicWidget">
         <property name="title">
          <string>Rotation</string>
         </property>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QCheckBox" name="rotationPCCheckBox">
         <property name="text">
          <string>hint / precomputed rotations</string>
         </property>
        </widget>
       </item>
       <item row="2" column="0">
        <widget class="QGroupBox" name="rotGroupBox">
         <property name="enabled">
          <bool>false</bool>
         </property>
         <property name="title">
          <string/>
         </property>
         <layout class="QGridLayout" name="gridLayout_5">
          <property name="margin">
           <number>6</number>
          </property>
          <property name="spacing">
           <number>5</number>
          </property>
          <item row="0" column="0">
           <widget class="QLabel" name="label_3">
            <property name="text">
             <string>Rot speed min:</string>
            </property>
           </widget>
          </item>
          <item row="0" column="1">
           <widget class="QDoubleSpinBox" name="rotSpeedMinDoubleSpinBox"/>
          </item>
          <item row="1" column="0">
           <widget class="QLabel" name="label_4">
            <property name="text">
             <string>Rot speed max:</string>
            </property>
           </widget>
          </item>
          <item row="1" column="1">
           <widget class="QDoubleSpinBox" name="rotSpeedMaxDoubleSpinBox"/>
          </item>
          <item row="2" column="0">
           <widget class="QLabel" name="label_5">
            <property name="text">
             <string>Nb models:</string>
            </property>
           </widget>
          </item>
          <item row="2" column="1">
           <widget class="QSpinBox" name="numModelsSpinBox">
            <property name="minimum">
             <number>1</number>
            </property>
            <property name="maximum">
             <number>32</number>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item row="3" column="0">
        <spacer name="verticalSpacer_3">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>17</width>
           <height>184</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="lookAtPage">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <attribute name="title">
       <string/>
      </attribute>
      <layout class="QGridLayout" name="gridLayout_6">
       <property name="margin">
        <number>6</number>
       </property>
       <property name="spacing">
        <number>3</number>
       </property>
       <item row="0" column="0">
        <widget class="QCheckBox" name="alignCheckBox">
         <property name="text">
          <string>Align on motion</string>
         </property>
        </widget>
       </item>
       <item row="2" column="0">
        <widget class="QLabel" name="blurCoeffLabel">
         <property name="text">
          <string>Fake motion blur coeff.:</string>
         </property>
        </widget>
       </item>
       <item row="3" column="0" colspan="3">
        <widget class="NLQT::CEditRangeFloatWidget" name="blurCoeffWidget" native="true"/>
       </item>
       <item row="4" column="0" colspan="2">
        <widget class="QLabel" name="blurTreshholdLabel">
         <property name="text">
          <string>Fake motion blur threshold:</string>
         </property>
        </widget>
       </item>
       <item row="5" column="0" colspan="3">
        <widget class="NLQT::CEditRangeFloatWidget" name="blurTresholdWidget" native="true"/>
       </item>
       <item row="6" column="1">
        <spacer name="verticalSpacer_4">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>203</height>
          </size>
         </property>
        </spacer>
       </item>
       <item row="1" column="0">
        <widget class="QCheckBox" name="zalignCheckBox">
         <property name="text">
          <string>Z-Align</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="shockWavePage">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <attribute name="title">
       <string/>
      </attribute>
      <layout class="QGridLayout" name="gridLayout_8">
       <property name="margin">
        <number>6</number>
       </property>
       <property name="spacing">
        <number>3</number>
       </property>
       <item row="0" column="0">
        <widget class="QLabel" name="label_6">
         <property name="text">
          <string>Radius cut:</string>
         </property>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="NLQT::CEditRangeFloatWidget" name="radiusCutWidget" native="true"/>
       </item>
       <item row="2" column="0">
        <widget class="QLabel" name="label_7">
         <property name="text">
          <string>Number of segments:</string>
         </property>
        </widget>
       </item>
       <item row="3" column="0">
        <widget class="NLQT::CEditRangeUIntWidget" name="shockWaveNbSegWidget" native="true"/>
       </item>
       <item row="4" column="0">
        <widget class="QLabel" name="label_8">
         <property name="text">
          <string>Texture U factor:</string>
         </property>
        </widget>
       </item>
       <item row="5" column="0">
        <widget class="NLQT::CEditRangeFloatWidget" name="shockWaveUFactorWidget" native="true"/>
       </item>
       <item row="6" column="0">
        <spacer name="verticalSpacer_5">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>251</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="fanLightPage">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <attribute name="title">
       <string/>
      </attribute>
      <layout class="QGridLayout" name="gridLayout_9">
       <property name="margin">
        <number>6</number>
       </property>
       <property name="spacing">
        <number>3</number>
       </property>
       <item row="0" column="0">
        <widget class="QLabel" name="label_9">
         <property name="text">
          <string>Number fan light:</string>
         </property>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="NLQT::CEditRangeUIntWidget" name="nbFanLightWidget" native="true"/>
       </item>
       <item row="2" column="0">
        <widget class="QLabel" name="label_10">
         <property name="text">
          <string>Phase smoothness:</string>
         </property>
        </widget>
       </item>
       <item row="3" column="0">
        <widget class="NLQT::CEditRangeUIntWidget" name="phaseSmoothnesWidget" native="true"/>
       </item>
       <item row="4" column="0">
        <widget class="QLabel" name="label_11">
         <property name="text">
          <string>Fan light speed:</string>
         </property>
        </widget>
       </item>
       <item row="5" column="0">
        <widget class="NLQT::CEditRangeFloatWidget" name="fanLightSpeedWidget" native="true"/>
       </item>
       <item row="6" column="0">
        <widget class="QLabel" name="label_12">
         <property name="text">
          <string>Fan light intensity:</string>
         </property>
        </widget>
       </item>
       <item row="7" column="0">
        <widget class="NLQT::CEditRangeFloatWidget" name="fanLightIntensityWidget" native="true"/>
       </item>
       <item row="8" column="0">
        <spacer name="verticalSpacer_6">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>229</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="ribbonTailPage">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <attribute name="title">
       <string/>
      </attribute>
      <layout class="QGridLayout" name="gridLayout_11">
       <property name="margin">
        <number>6</number>
       </property>
       <property name="spacing">
        <number>3</number>
       </property>
       <item row="0" column="0">
        <widget class="QLabel" name="label">
         <property name="text">
          <string>Number of segments:</string>
         </property>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="NLQT::CEditRangeUIntWidget" name="tailNbSegsWidget" native="true"/>
       </item>
       <item row="2" column="0">
        <widget class="QLabel" name="label_14">
         <property name="text">
          <string>Duration segment:</string>
         </property>
        </widget>
       </item>
       <item row="3" column="0">
        <widget class="NLQT::CEditRangeFloatWidget" name="segDurationWidget" native="true"/>
       </item>
       <item row="4" column="0">
        <widget class="QLabel" name="tailTexUflabel">
         <property name="text">
          <string>Texture U factor:</string>
         </property>
        </widget>
       </item>
       <item row="5" column="0">
        <widget class="NLQT::CEditRangeFloatWidget" name="ribbonTexUfactorWidget" native="true"/>
       </item>
       <item row="6" column="0">
        <widget class="QLabel" name="tailTexVflabel">
         <property name="text">
          <string>Texture V factor:</string>
         </property>
        </widget>
       </item>
       <item row="7" column="0">
        <widget class="NLQT::CEditRangeFloatWidget" name="ribbonTexVfactorWidget" native="true"/>
       </item>
       <item row="8" column="0">
        <widget class="QGroupBox" name="ribbonGroupBox">
         <property name="title">
          <string/>
         </property>
         <layout class="QGridLayout" name="gridLayout">
          <property name="margin">
           <number>6</number>
          </property>
          <property name="spacing">
           <number>3</number>
          </property>
          <item row="0" column="0">
           <widget class="QCheckBox" name="constantLengthCheckBox">
            <property name="toolTip">
             <string>Non parametric anim only</string>
            </property>
            <property name="text">
             <string>Force constant length(m):</string>
            </property>
           </widget>
          </item>
          <item row="1" column="0">
           <widget class="NLQT::CEditRangeFloatWidget" name="ribbonLengthWidget" native="true">
            <property name="enabled">
             <bool>false</bool>
            </property>
           </widget>
          </item>
          <item row="2" column="0">
           <widget class="QLabel" name="lodDegradationLabel">
            <property name="text">
             <string>LOD degradation:</string>
            </property>
           </widget>
          </item>
          <item row="3" column="0">
           <widget class="NLQT::CEditRangeFloatWidget" name="lodDegradationWidget" native="true"/>
          </item>
          <item row="4" column="0">
           <widget class="QLabel" name="label_15">
            <property name="text">
             <string>Trail coord. system:</string>
            </property>
           </widget>
          </item>
          <item row="5" column="0">
           <widget class="QComboBox" name="coordSystemComboBox">
            <item>
             <property name="text">
              <string>Local</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>World</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>User</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>Father coord. system</string>
             </property>
            </item>
           </widget>
          </item>
          <item row="6" column="0">
           <widget class="QCheckBox" name="useHermitteCheckBox">
            <property name="text">
             <string>Use hermitte interpolation</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item row="9" column="0">
        <widget class="NLQT::CTailParticleWidget" name="tailWidget" native="true"/>
       </item>
       <item row="10" column="0">
        <spacer name="verticalSpacer_7">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>100</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="meshPage">
      <attribute name="title">
       <string/>
      </attribute>
      <layout class="QGridLayout" name="gridLayout_12">
       <property name="margin">
        <number>6</number>
       </property>
       <property name="spacing">
        <number>3</number>
       </property>
       <item row="0" column="0">
        <widget class="NLQT::CMeshWidget" name="meshWidget" native="true"/>
       </item>
       <item row="1" column="0">
        <widget class="NLQT::CConstraintMeshWidget" name="constraintMeshWidget" native="true"/>
       </item>
       <item row="2" column="0">
        <spacer name="verticalSpacer_8">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>407</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="texturePage">
      <attribute name="title">
       <string/>
      </attribute>
      <layout class="QGridLayout" name="gridLayout_13">
       <property name="margin">
        <number>6</number>
       </property>
       <property name="spacing">
        <number>3</number>
       </property>
       <item row="0" column="0">
        <widget class="NLQT::CParticleTextureWidget" name="particleTextureWidget" native="true"/>
       </item>
       <item row="1" column="0">
        <widget class="NLQT::CParticleTextureAnimWidget" name="texAnimWidget" native="true"/>
       </item>
       <item row="2" column="0">
        <spacer name="verticalSpacer_90">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeType">
          <enum>QSizePolicy::Expanding</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>463</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>NLQT::CParticleTextureWidget</class>
   <extends>QWidget</extends>
   <header>particle_texture_widget.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>NLQT::CEditRangeUIntWidget</class>
   <extends>QWidget</extends>
   <header>edit_range_widget.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>NLQT::CEditRangeFloatWidget</class>
   <extends>QWidget</extends>
   <header>edit_range_widget.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>NLQT::CAttribFloatWidget</class>
   <extends>QGroupBox</extends>
   <header>attrib_widget.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>NLQT::CMeshWidget</class>
   <extends>QWidget</extends>
   <header>mesh_widget.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>NLQT::CConstraintMeshWidget</class>
   <extends>QWidget</extends>
   <header>constraint_mesh_widget.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>NLQT::CTailParticleWidget</class>
   <extends>QWidget</extends>
   <header>tail_particle_widget.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>NLQT::CParticleTextureAnimWidget</class>
   <extends>QWidget</extends>
   <header>particle_texture_anim_widget.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>NLQT::CAttribRGBAWidget</class>
   <extends>QGroupBox</extends>
   <header>attrib_widget.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>NLQT::CAttribPlaneBasisWidget</class>
   <extends>QGroupBox</extends>
   <header>attrib_widget.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
